src/api/spec/models/report_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe Report do
+  describe '#reports_pointing_to_same_reportable' do
+    context 'when different reportables' do
+      let(:decision) { create(:decision) }
+      let(:report) { build(:report, decision: decision) }
+
+      before do
+        report.save
+      end
+
+      it { expect(report.valid?).to be(false) }
+      it { expect(report.errors.full_messages).to eq(['Decision has reports pointing to a different reportable. All decision reports should point to same reportable.']) }
+    end
+
+    context 'when reports are pointing to same trportable' do
```

Comment:
There is a small typo here: `trportable` instead of `reportable`

Reply:
[Yes, sir](https://github.com/openSUSE/open-build-service/pull/15389)!

Reply:
Thank you :slightly_smiling_face: